docs/src/main/java/docs/ddata/protobuf/msg/TwoPhaseSetMessages.java

```diff
@@ -18,81 +18,87 @@
 
 public final class TwoPhaseSetMessages {
   private TwoPhaseSetMessages() {}
+
   public static void registerAllExtensions(
-      org.apache.pekko.protobufv3.internal.ExtensionRegistryLite registry) {
-  }
+      org.apache.pekko.protobufv3.internal.ExtensionRegistryLite registry) {}
 
   public static void registerAllExtensions(
       org.apache.pekko.protobufv3.internal.ExtensionRegistry registry) {
-    registerAllExtensions(
-        (org.apache.pekko.protobufv3.internal.ExtensionRegistryLite) registry);
+    registerAllExtensions((org.apache.pekko.protobufv3.internal.ExtensionRegistryLite) registry);
   }
-  public interface TwoPhaseSetOrBuilder extends
+
+  public interface TwoPhaseSetOrBuilder
+      extends
```

Comment:
Do we need the code format changes?

Reply:
the PR is big enough without making us have to read through all the unrelated code reformatting

Reply:
I was run javafmtall，I CAN revert this tomorrow

Reply:
if you like, you could do a separate PR that does javafmtAll - we merge that PR and then you can rebase this PR. Is that ok?

Reply:
I have finally remove these files from the PR :(